src/dispatchers/mouseDispatcher.ts

```diff
@@ -178,11 +178,13 @@ export module Dispatchers {
      * Computes the mouse position from the given event, and if successful
      * calls all the callbacks in the provided callbackSet.
      */
-    private _measureAndDispatch(event: MouseEvent, callbackSet: Utils.CallbackSet<MouseCallback>) {
-      var newMousePosition = this._translator.computePosition(event.clientX, event.clientY);
-      if (newMousePosition != null) {
-        this._lastMousePosition = newMousePosition;
-        callbackSet.callCallbacks(this.lastMousePosition(), event);
+    private _measureAndDispatch(event: MouseEvent, callbackSet: Utils.CallbackSet<MouseCallback>, captureAllEvents: boolean = false) {
```

Comment:
Please don't use boolean parameters in methods.

http://blogs.msdn.com/b/oldnewthing/archive/2006/08/28/728349.aspx